uplink/builder.py

```diff
@@ -128,8 +135,11 @@ def _bind_to_instance(instance, attribute_name, call_factory):
         setattr(instance, attribute_name, call_factory)
 
     def build(self, *args, **kwargs):
```

Comment:
It seems that the `*args` and `*kwargs` can be removed now, since we're not constructing the consumer here anymore.